_posts/2021-06-10-external-domain-migration-announcement.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Migrate to the external-domain service"
+date: June 10, 2021
+excerpt: Customers need to take action to migrate from the cdn-service and custom-domain service to the external-domain service
+---
+
+## Who is this for?
+
+This post is for all users of the cdn-route or custom-domain services.
```

Comment:
Should we include a specific qualifier here so people know these are deprecated services and to remind them?  e.g.:

```suggestion
This post is for all users of the deprecated cdn-route or custom-domain services.
```